### ISSUE TITLE
Allow changing download destination path. 

### ISSUE THREAD

**Issue by recolic** [nextcloud/android#7128]
### Is your feature request related to a problem? Please describe.

Sometimes I download a picture from nextcloud, and would like to share it in another app. 

usually, downloaded files are expected to be in `/sdcard/Downloads`. However, I realized that my picture ended in `/external_files/emulated/0/Android/media/com.nextcloud.client/nextcloud/recolic@driver.recolic.cc/tmp/Screenshot.png`. The path is too complicated, and it's annoying to open so many folder to use it. 

### Describe the solution you'd like

There should be an configurable option "`Path for downloaded file`" in settings. All downloaded files should be there, WITHOUT preserving relative path `DRIVE_ROOT/tmp/Screenshot.png`. 

all medias are file. We should treat all files equally, and put everything into one single `Downloaded` dir, **just like what chrome and firefox do**. Simple is beautiful. 

### Describe alternatives you've considered
Manually open 10 directories on every usage.

**Comment by tobiasKaminsky** [comment]
If you download a file it is kept in NC folder so that it can be synced again.
If you want to share a file, open NC and click on share button.

If you would download (which is currently not supported) a file to e.g. /sdcard/ then it is out of sync.
(same like if you would copy a file out of NC folder on desktop…)

**Comment by recolic** [comment]
Yeah... Sometimes I need to compress/decompress a zip file, edit the png with Google Photos, recognize the png with Google Lens... 

If the file could be easily accessed from traditional File Explorer, some tasks may be easier..

**Comment by tobiasKaminsky** [comment]
True, I just wanted to point out that "save to sdcard" then means that files are out of sync and thus might create other people to complain on it (and it is another menu entry point).

As far as I know this is the way e.g. Dropbox/Drive are doing this:
One can only export/save it, but not directly access the synced files

**Comment by recolic** [comment]
Telegram has two different options for media files: "Save to gallery" 
and "Save to downloads". I think that's really easy to use...

Maybe we can have something like: "Save to local drive", and "Export to 
downloads"

**Comment by danrneal** [comment]
Perhaps there could be two options?  

* "Download" which will download a copy to the downloads folder (out of sync file)
* "Available offline" which will keep a local copy on the phone in the obscure folder (in sync)

**Comment by joshtrichards** [comment]
This now the case today, using the *Export* option. 
Duplicate of #9519 
Fixed in #10443
